.build/build-owasp.xml

```diff
@@ -83,4 +85,67 @@
             </fileset>
         </dependency-check>
     </target>
+
+    <target depends="init" name="license-report">
+        <exec executable="jq" osfamily="unix" dir="${basedir}" logError="true" failonerror="false" failifexecutionfails="false">
```

Comment:
@jacek-lewandowski  @michaelsembwever  This is suspicious. So we need to have `jq` locally, correct? I think we need be very picky and explicit about this. Until now, all you need is git, ant, python and shell.

Reply:
we might have a check for this in `build.xml` as we have for Ant.

    <fail message="You need to use Ant of version at least 1.10 to continue.">
      <condition>
        <not>
          <antversion atleast="1.10"/>
        </not>
      </condition>
    </fail>

Even "antversion" is in-built target (I think so), something similar should be done for all other needed tooling and error out as soon as possible if it is not available so it will not fail unnecessarily somewhere down the road.

Reply:
I see there is `failifexecutionfails="false", could you explain the consequences? So we are OK when this fails? So if that command is not present, we will just silently skip it? What are the consequences of that?

Reply:
ok, I realized the same and refactored it to run with JavaScript (no worries about j17). Now it seems to be better / cleaner from the user pov.